xcpmd/src/default.rules

```diff
@@ -0,0 +1,15 @@
+# Default set of rules for XCPMD. These are only loaded when no rules or vars
+# exist in the DB.
+
+# Variable section
+battCritical(10)
+criticalBrightness(60)
+brightnessOnBatt(75)
+=
```

Comment:
Pretty minor, but at first sight, this looks out of place.
Maybe a `# Separator` comment would help.